website/pages/docs/install.mdx

```diff
@@ -5,9 +5,15 @@ description: 'How to install Million.js into your React project'
 
 import { Callout, Tabs, Tab, Steps } from 'nextra-theme-docs';
 
+
 # Installation
 
-Million.js assumes that you've already initialized a React project. If you haven't, we recommend checking out [react.dev](https://react.dev) first.
+Million.js assumes that you've already initialized a React project. If you haven't, it is recommended you check out [react.dev](https://react.dev) first.
```

Comment:
This is probably more appropriate for an aside.

Reply:
Hmm, thanks for the note. For now, I can try changing the structure to match what you suggested first, I'll reach out to Aiden for his thoughts on it. 🙌.